spec/controllers/educations_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe EducationsController, type: :controller do
+
+  describe 'GET #index' do
+    # it "populates an array of all educations" do
+    #   monte = create(:education, school: 'Monte')
+    #   ssori = create(:education, school: 'ssori')
+    #   get :index
+    #   expect(assigns(:education)).to match_array([monte, ssori])
+    # end
```

Comment:
Oh, I thought it was finished since there was a pull request. You can always shoot people a link to your branch or specific commit if they're interested in learning. You can also preface the title of the pull request with WIP (work in progress). That will signify that it's not done yet